conformance/fhir-ig-davinci-pdex/src/main/resources/hl7/fhir/us/davinci-pdex/package/.index.json

```diff
@@ -6,21 +6,21 @@
       "resourceType": "ImplementationGuide",
       "id": "hl7.fhir.us.davinci-pdex",
       "url": "http://hl7.org/fhir/us/davinci-pdex/ImplementationGuide/hl7.fhir.us.davinci-pdex",
-      "version": "1.0.0"
+      "version": "2.0.0-ballot"
```

Comment:
I think we want to keep 1.0.0 around and add 2.0.0 as a new package
1. move the 1.0.0 stuff from hl7/fhir/us/davinci-pdex/ to hl7/fhir/us/davinci-pdex/100
2. add the new 2.0.0-ballot stuff in a new package hl7/fhir/us/davinci-pdex/200

Reply:
OK, I think I'll start over with a new branch so it's easier to review how the current files went to hl7/fhir/us/davinci-pdex/100.  Closing this pull request and deleting the branch.

Reply:
Done in https://github.com/IBM/FHIR/pull/3424.